### ISSUE TITLE
Have tree-less tab bar replacement view

### ISSUE THREAD

**Issue by BugZappr** [cxw42/TabFern#95]
I'd just like to use TF as a tab bar replacement.  I imagine that TF could have an option to live as a sort of replacement tab bar for Google's awful tab bar.  I keep it open in a column to the left of a single browser window.  The top-level tree view takes up valuable pixels in a vertical bar to the left of the tab.  I think TF might just only show the tabs in the active window, changing them when another gets activated.  True, it might not always be to the left of the newly activated window, but I can resize TF and the windows so that TF is always in a little vertical bar to the left of my active window.

**Comment by cxw42** [comment]
Thanks for the request!  Do you want any saving of tabs, or just the vertical display per window?  If the latter, I would recommend a separate extension.  Snapping the tab list to the open window is not too hard.  It would be nicer if Chrome supported sidebars, of course, but as far as I know it does not.

**Comment by cxw42** [comment]
See further discussion in #93

**Comment by BugZappr** [comment]
It could be a separate extension.  There are many like it.  The existing ones work poorly.  One argument for not making it a separate extension, is that you wouldn't have to duplicate much of your code-base in another project to maintain; but in either case, I am suggesting a per-window vertical tabs, except one that works.  If you think of the now lost vertical tabs in Chrome, and the existing one in Vivaldi, I'd say they are evidently per-window.

**Comment by cxw42** [comment]
I will give this some thought.  The point about duplication is well made, but will be less of an issue after #4.  I also note that TabFern doesn't have URL-as-tooltip - feel free to open another issue of that's of interest to you in TF as it is now.  Thanks!
